copy/multiple.go

```diff
@@ -29,8 +29,9 @@ const (
 )
 
 type instanceCopy struct {
-	op           instanceCopyKind
-	sourceDigest digest.Digest
+	op                     instanceCopyKind
+	sourceDigest           digest.Digest
+	forceCompressionFormat bool
```

Comment:
If this is relevant only to `instanceCopyCopy`, name it `copy…` and move it into a separate section.

---

Non-blocking: again, it seems more elegant to me to for have a `options copySingleImageOptions` field here, set with _all_ the options, shared for both `…Copy` and `…Clone`, and trivially applied in the execution loop.